### PR TITLE
All pages headers

### PR DESCRIPTION
**Pull request by Samah-Mustafa**

### CODE REVIEW

**Comment by Samah-Mustafa** [comment]
@louisrli PTAL
This PR is more bigger than the others because I made changes on all headers and footers and styles for both of  them then all of that added to the main file in each directory

**Review by basil-kawak — COMMENTED** [review]
looks good

File: src/components/common/NavBar.jsx [hunk 1]
```diff
@@ -5,62 +5,94 @@ import { Link } from "react-router-dom";
 import LangDropdown from "./LangDropdown";
 
 export const NavBar = () => {
+  const [isHamburgerShown, setIsHamburgerShown] = React.useState("-100%");
+
+  const handlehamburger = () => {
```
Comment (louisrli) [inline src/components/common/NavBar.jsx]: this branch has files from the other PR, please clean it up so i can review the actual changes

**Comment by Samah-Mustafa** [comment]
@louisrli PTAL

**Comment by louisrli** [comment]
fix test please

**Comment by Samah-Mustafa** [comment]
@louisrli Actually we are all working on it since yesterday

**Comment by louisrli** [comment]
github says test is failing, it should be fixed before requesting review

**Comment by Samah-Mustafa** [comment]
@louisrli Yes, I'm sorry for asking you to review before passing it

**Comment by Samah-Mustafa** [comment]
@louisrli PTAL

**Review by louisrli — APPROVED** [review]

**Review by basil-kawak — APPROVED** [review]

**Pull request merged**